package.json

```diff
@@ -23,7 +23,7 @@
     "socks5"
   ],
   "engines": {
-    "node": ">= 10.13.0",
+    "node": ">= 16.0.0",
```

Comment:
dropping support for node version is a breaking change - it looks like `ip-address` v7 supports Node v10; would it be possible to use that instead?

Reply:
I think i'll just bump the overall package to 3.0.0. Anything prior to node 16.x isn't being maintained at this point.

Reply:
that'll mean downstream packages will need to explicitly update their dependency on `sock` which could delay rollout - any chance of getting a final v2 patch with this before immediately releasing v3?

Reply:
btw If you're running into problems related to Node 10, I'm happy to have a crack at getting things working :)

Reply:
I think it may just be the typescript building not working on 10.x, the actual javascript may work fine tbh.

Reply:
Okay, yeah so it works just fine on plain javascript node 10.24.1 with ip-address 9.x, it's just the building of the typescript/etc won't work for anything less than 16.

Reply:
So I'll just leave the min version as-is with 10.x, and release as 2.8.0? Or should this just be a bug/patch as 2.7.2?

Reply:
I think technically it's a feature but I don't think anyone would complain if you released it as a bug fix 🤷